pkg/kubelet/config/config.go

```diff
@@ -94,7 +94,7 @@ func (c *PodConfig) SeenAllSources(seenSources sets.String) bool {
 	if c.pods == nil {
 		return false
 	}
-	klog.V(5).Infof("Looking for %v, have seen %v", c.sources.List(), seenSources)
+	klog.V(5).InfoS("Looking for list of sources, have seen source ", "sourceList", c.sources.List(), "seenSources", seenSources)
```

Comment:
```suggestion
	klog.V(5).InfoS("Looking for list of sources, have seen source ", "sources", c.sources.List(), "seenSources", seenSources)
```

Reply:
suggested change conflict with one above (sources vs sourceList), but both agree on text ("Looking for list of sources") which is currently different ("Looking for sources")